libafl/src/schedulers/weighted.rs

```diff
@@ -340,7 +340,7 @@ where
             };
 
             // Update depth
-            if current_cycles > corpus_counts {
+            if runs_in_current_cycle == corpus_counts {
```

Comment:
i think >= is better? addison added that check because corpus_counts could actually decrease..

Reply:
what i mean is that you can get corpus_counts decreased by N so runs_in_current_cycle is not necessarily equal to corpus count but you still have to update it.

Reply:
Ah, makes sense.